api.docker

```diff
@@ -2,9 +2,9 @@ FROM digitalmarketplace/base
 
 COPY nginx/api /etc/nginx/sites-enabled/api
 
-ONBUILD RUN python -m venv venv
+ONBUILD RUN /usr/local/bin/python3 -m venv venv
 ONBUILD COPY requirements.txt ${APP_DIR}
-ONBUILD RUN venv/bin/pip install --no-cache-dir -r requirements.txt
+ONBUILD RUN /app/venv/bin/pip3 install --no-cache-dir -r requirements.txt
 
 ONBUILD COPY . ${APP_DIR}
```

Comment:
This seems to also rely on the current directory, so for consistency should we also substitute `/app` for `.`?